<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1764</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>1701</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>타임스탬프</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>등록 번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고(고객메모)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고(직원메모)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주문번호(ESM)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>제품사</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주문 상태</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>설치 요청일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>설치 완료일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>모델명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>렌탈료</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>색상</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계약자</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연락처</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="win_modify_complete">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>50</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>등록 완료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_modify_install_complete">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>설치 완료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_data_modify">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>170</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>데이터 수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_find_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검 색</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>모두</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RUHENS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HYUNDAI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUCKOO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CHUNGHO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KYOWON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BODYFRIEDN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WELRIX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NOVITA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SAMSUNG</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="win_modify_cancel">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>272</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>설치 완료 날짜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_reconfirm_btn">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>재확인 요청</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="win_cancel_reason">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="time_start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>272</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="time_end">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>272</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 타임 스태프 시작 날짜 (기본 100일 전)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 타임 스태프 마지막 날짜 </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>등록 완료</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>설치 보류</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>설치 완료</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>취소</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>재확인 요청</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>모두</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>브랜드</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>주문 상태</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="win_reconfirm_reason">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>10</y>
     <width>21</width>
     <height>231</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
